gensim/models/nmf.py

```diff
@@ -0,0 +1,193 @@
+from itertools import chain
+
+import numpy as np
+from scipy.stats import halfnorm
+
+
+class NMF(object):
```

Comment:
Have a look at my previous comments (you didn't fix part of them https://github.com/RaRe-Technologies/gensim/pull/2007#pullrequestreview-108027636)

Reply:
Fixed